forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -2154,6 +2154,10 @@ public static int xCount(final Card c, final String s, final CardTraitBase ctb)
             return doXMath(c.getRememberedCount(), expr, c, ctb);
         }
 
+        if (sq[0].startsWith("ChosenSize")) {
```

Comment:
I will make use of this in Spara's Bodyguard

<img width="759" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/457c464d-e52f-46a8-b503-ac38f00aadc1">